dartsim/src/SDFFeatures.cc

```diff
@@ -117,7 +117,6 @@ static void CopyStandardJointAxisProperties(
     const int _index, Properties &_properties,
     const ::sdf::JointAxis *_sdfAxis)
 {
-  _properties.mInitialPositions[_index] = _sdfAxis->InitialPosition();
```

Comment:
this is deprecated but still functional; so I would keep this line and suppress its deprecation warnings (currently there are test failures because this was removed)

Reply:
The test has been fixed. I followed the approach being taken on https://github.com/ignitionrobotics/ign-physics/pull/189/.

I can suppress the warning instead if that's preferred. I'm not sure if there's a new approach to use or if this feature will just disappear when we use `SDF 12`. SDFormat's migration guide didn't list any replacements. Are we sure we want to remove the feature?

CC @azeey

Reply:
I think the replacement may be using the `<state>` element, but until we've articulated that I'd like to keep the feature working, since it is only deprecated at this point, not disabled. I'll add a comment to that effect on #189 as well.

Reply:
> I'll add a comment to that effect on #189 as well.

https://github.com/ignitionrobotics/ign-physics/pull/189/files#r558691120

Reply:
macro to suppress deprecation warnings in https://github.com/ignitionrobotics/ign-utils/pull/5

Reply:
Suppressing the warning in a more traditional style while this library doesn't depend on `ign-utils` yet. I'd like to merge this ASAP to unblock upstream.

35ce9275e3a60077dab35a5a61f8050478abac72